packages/web/scripts/copyCSSFile.js

```diff
@@ -1,3 +1,3 @@
-const fs = require('fs');
+import { copyFileSync } from 'fs';
```

Comment:
I receive a ` Cannot use import statement outside a module` error when running `npm run build` with these changes.

Reply:
Ah damn, that's too bad - I'll revert that change for now.